orchestra/workflow/load.py

```diff
@@ -11,6 +12,20 @@
 from orchestra.core.errors import WorkflowError
 
 
+def get_workflow_version_slugs():
+    versions = {}
+    for app_name in settings.ORCHESTRA_WORKFLOWS:
+        # App label is the last part of the app name by default
+        app_label = app_name.split('.')[-1]
```

Comment:
if two apps have the same workflow slug, we should raise

then this thing would just return {workflow_slug: [version, slugs]}